www/templates/main-metrics.html

```diff
@@ -0,0 +1,60 @@
+<ion-view view-title="Metrics">
+  <ion-nav-bar class="bar-stable">
+    <ion-nav-back-button></ion-nav-back-button>
+    <ion-nav-buttons side="right">
+      <button class="button button-assertive button-block"
+              ng-click="getMetrics()">Get!</button>
```

Comment:
I tried to have it as part of the content back when I made the heatmap screen, but the problem is that everything else comes in twos while the button is single. Putting it in the content creates a new col which is a problem when we already have very little width.

Maybe we could move it under the metric name? Anyway, my goal was to create a functional and not too ugly UI.

My hope is that now some of the you can work on the design - we will reuse it for heatmap, metrics and potentially for other searches (trips, common trips) in the future.